README.md

```diff
@@ -27,6 +27,76 @@ https://eui-event.vercel.app/
 - Run `yarn start dev` to start it up!
 - Go to localhost:3000 in your browser to view it!
 
+## Connect Google Sheets with your app
+
+Follow these steps to connect your google sheets file with the app.
+
+### 1. Create a google sheet
+
+#### 1. Go to https://docs.google.com/ and create a blank sheet
```

Comment:
Should we make a recommendation for what they should name the sheet?

Reply:
Actually... what if we made a template google sheet, like this : and we just offered for them to copy the sheet and rename it? https://docs.google.com/spreadsheets/d/1OKYzfn9oKqJHJphsipYj_bsPvcjqAjPr2NRjihkKlqg/edit?usp=sharing

Reply:
Noted. I will add a link to the example sheet that I have used in the demo video.